pages.es/linux/ufw.md

```diff
@@ -0,0 +1,37 @@
+# ufw
+
+> Cortafuegos sin complicaciones (Uncomplicated Firewall).
```

Comment:
```suggestion
> Cortafuegos sin complicaciones (_Uncomplicated Firewall_).
```

Reply:
Why the underscores?

Reply:
Because these are an English words. it's a common usage.

Reply:
Is this indented to be italic? Because I don't think most clients support that.

Reply:
If it's not supported the words are still readable. pages like `supervisorctl` and `csvcut` have `**` and we have no issues with them.